cohort_banking_replicator_js/package.json

```diff
@@ -17,7 +17,7 @@
   },
   "dependencies": {
     "@types/pg": "^8.10.2",
-    "cohort_sdk_js": "file:../packages/cohort_sdk_js",
+    "cohort_sdk_client": "file:../cohort_sdk_client",
     "nodejs-dashboard": "^0.5.1",
     "pg": "^8.11.3",
     "ts-node": "^10.9.1",
```

Comment:
do we need `ts-node`? Since you use `tsc` to transpile `ts` to `js`, I think we can remove this.

Reply:
You are right this can be removed.